packs/github/actions/lib/base.py

```diff
@@ -16,11 +16,12 @@ def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('base_url', None)
```

Comment:
Should probably use the current default as "https://github.com" instead of None.